src/gazebo_gps_plugin.cpp

```diff
@@ -38,6 +38,20 @@ GpsPlugin::~GpsPlugin()
       updateConnection_->~Connection();
 }
 
+bool GpsPlugin::checkWorldHomePosition(physics::WorldPtr world) {
+  if(!world_->SphericalCoords()) {
+    return false;
+  }
+  double latitude = world_->SphericalCoords()->LatitudeReference().Degree();
+  double longitude = world_->SphericalCoords()->LongitudeReference().Degree();
+  double altitude = world_->SphericalCoords()->GetElevationReference();
+  if(latitude || longitude || altitude ) {
```

Comment:
Shouldn't we make them all requires?

Reply:
I was assuming if either one of them are non-zero, it is valid. I am not sure if this is a valid assumption. Should we require it to be all non-zero?

Reply:
I think it is a better conceptual choice to consider all required.

Reply:
```suggestion
  if (latitude && longitude && altitude) {
```

Reply:
Makes sense